core/src/main/java/org/infinispan/marshall/exts/CacheRpcCommandExternalizer.java

```diff
@@ -155,22 +154,7 @@ public Integer getId() {
    }
 
    private StreamingMarshaller getCacheMarshaller(ByteString cacheName) {
```

Comment:
Actually, there's a few more opportunities to simplify this and maybe the MarshalledValue externalizer. Great spot!